Sources/RInAppMessaging/RInAppMessaging.swift

```diff
@@ -211,6 +214,17 @@ import RSDKUtils
         return configURL
     }
 
+    /// Checks the existence of RMC module and verifies the `configure()` caller.
+    /// - Parameter caller: a path to the source file that called `configure()`
+    /// - Returns: `false` if RMC module is integrated and the method wasn't called from the RMC module.
+    private static func verifyRMCEnvironment(caller: String) -> Bool {
```

Comment:
Please make it `internal static func` without any function param so that we can use this func for SDKCF-6709